centreon/packages/ui/src/Graph/BarStack/BarStack.styles.ts

```diff
@@ -16,7 +16,7 @@ export const useBarStackStyles = makeStyles()((theme) => ({
   svgContainer: {
     alignItems: 'center',
     backgroundColor: theme.palette.background.panelGroups,
-    borderRadius: theme.shape.borderRadius,
+    borderRadius: theme.spacing(1.25),
```

Comment:
Keep using borderRadius

Reply:
The border radius in theme.shape.borderRadius doesn't match up with what's in the prototype, so they don't look the same.